cyhy_report/customer/report.mustache

```diff
@@ -730,7 +730,7 @@ As you review the report, you may have additional questions. Check out the answe
      \renewcommand{\arraystretch}{1.8}  % Increase vertical padding between rows
       \begin{tabular}{clccl}
         \raisebox{-0.55\height}{\scalegraphicsnofig[1.35]{assets/<<^risky_services.rdp.count>>green-circle<</risky_services.rdp.count>><<#risky_services.rdp.count>><<#risky_services.rdp.any_newly_opened>>red-triangle<</risky_services.rdp.any_newly_opened>><<^risky_services.rdp.any_newly_opened>>blue-square<</risky_services.rdp.any_newly_opened>><</risky_services.rdp.count>>}} & \vtop{\hbox{\strut {\textcolor{rc-dark-blue}{\RCSourceSansProTwelve{\textbf{RDP*}}}}} \hbox{\strut {\textcolor{rc-dark-blue}{\RCSourceSansProTwelve{\numprint{<<&risky_services.rdp.count>>}}}}}} & \ \ \ \ \
-        & \raisebox{-0.55\height}{\scalegraphicsnofig[1.35]{assets/<<^risky_services.ftp.count>>green-circle<</risky_services.ftp.count>><<#risky_services.ftp.count>><<#risky_services.ftp.any_newly_opened>>red-triangle<</risky_services.ftp.any_newly_opened>><<^risky_services.ftp.any_newly_opened>>blue-square<</risky_services.ftp.any_newly_opened>><</risky_services.ftp.count>>}} & \vtop{\hbox{\strut {\textcolor{rc-dark-blue}{\RCSourceSansProTwelve{\textbf{FTP*}}}}} \hbox{\strut {\textcolor{rc-dark-blue}{\RCSourceSansProTwelve{\numprint{<<&risky_services.ftp.count>>}}}}}} \\
+        & \raisebox{-0.55\height}{\scalegraphicsnofig[1.35]{assets/<<^risky_services.ftp.count>>green-circle<</risky_services.ftp.count>><<#risky_services.ftp.count>><<#risky_services.ftp.any_newly_opened>>red-triangle<</risky_services.ftp.any_newly_opened>><<^risky_services.ftp.any_newly_opened>>blue-square<</risky_services.ftp.any_newly_opened>><</risky_services.ftp.count>>}} & \vtop{\hbox{\strut {\textcolor{rc-dark-blue}{\RCSourceSansProTwelve{\textbf{FTP}}}}} \hbox{\strut {\textcolor{rc-dark-blue}{\RCSourceSansProTwelve{\numprint{<<&risky_services.ftp.count>>}}}}}} \\
```

Comment:
Why was the asterisk removed here?  (This may be a dumb question, since I am quite unfamiliar with these risky services changes.)

Reply:
@jsf9k the report card had FTP listed and had an asterisk next to it. Cyber Directives pulled FTP from BOD 23-02 scope. Is this a case where we should have left the asterisks? There is a full description of why the change was requested on the issue I submitted. Does this help?

Reply:
I'm following the plot now.  Thank you.

Reply:
Sweet! And no thank you for the help @jsf9k